pkg/middlewares/aws_route_table_expander.go

```diff
@@ -80,6 +80,25 @@ func (m *AwsRouteTableExpander) handleTable(table *aws.AwsRouteTable, results *[
 			m.alerter.SendAlert(aws.AwsRouteTableResourceType, newInvalidRouteAlert(aws.AwsRouteTableResourceType, table.Id))
 			continue
 		}
+
+		routeAlreadyExists := false
```

Comment:
You can put this code (check if the route exist in a slice of resources) in a unexported struct `AwsRouteTableExpander` method to reuse logic for route and default tables.